ciao-controller/instance.go

```diff
@@ -46,7 +46,7 @@ type instance struct {
 	startTime time.Time
 }
 
-func isCNCIWorkload(workload *types.Workload) bool {
+func isCNCIWorkload(workload types.Workload) bool {
```

Comment:
@markdryan ok, thanks for clarifying. I did have another reason as well - and that was that I was trying to reduce the likelyhood that a receiver would modify the struct. It felt safer to send a copy.